feat(box-shadow): Add tokens

alexandra-lim opened the pull request:
This MR adds the box-shadow tokens to our token repository!

[Light theme shadow tokens](https://www.figma.com/file/784YociOk102q8PgCz0xAl/Design-Tokens?type=design&node-id=2388%3A5111&t=5US7D96Qn2PrY0eE-1)
[Dark theme shadow tokens](https://www.figma.com/file/lErKyeOWjHM6yiMsY5haos/Design-Tokens-(Dark)?type=design&node-id=2326%3A4815&t=cXhfolBXjlbcPJho-1)

alexandra-lim commented on src/data/emerson.json:
```diff
@@ -687,6 +687,69 @@
+    },
+    "shadow": {
+      "base": {
+        "value": "{colors.shades.black}",
```
Don't believe we have an official colour set for Emerson yet. Just a placeholder value

CCAyl commented on dist/tokens/native/themes/light.d.ts:
```diff
@@ -15,4 +15,5 @@ export const highlight : {textBase : string,textHover : string,textActive : stri
-export const light : {backgroundBase : string,backgroundContrast : string,backgroundHover : string,backgroundActive : string,textBase : string,linkMutedBase : string,linkMutedHover : string};
\ No newline at end of file
+export const light : {backgroundBase : string,backgroundContrast : string,backgroundHover : string,backgroundActive : string,textBase : string,linkMutedBase : string,linkMutedHover : string};
+export const shadow : {base : string};export const boxShadows: {};
```
we may need to noodle on our transforms again 🤔

alexandra-lim commented:
Fixed the issue thanks to Cath's help!

The SCSS files don't have the shadow tokens yet, but I've added a ticket in Gitlab to track that: https://git.fullscript.io/developers/hw-admin/-/issues/32385

iryanclarke commented on dist/documentation/themes/dark.js:
```diff
@@ -502,4 +502,9 @@ light : {backgroundBase:{
             hex: "#FFFFFF",
             description: "Used for link hover colour for light profile components on contrasting surface backgrounds"
+          }},
+shadow : {base:{
```
🍅 🍅 Will we pluralize this once we add more?

alexandra-lim replied:
I don't see us adding more since all the shadows are using the same base colour, just changes for each theme

iryanclarke commented on dist/documentation/themes/light.js:
```diff
@@ -502,4 +502,9 @@ light : {backgroundBase:{
+shadow : {base:{
+            name: "shadow.base",
+            hex: "#36485C",
+            description: "For use only with box shadow tokens"
```
Are we not parsing the `boxShadows` for use within themes? Or was your thought we would import the shadows seperately (aka `import { shadows } from "aviary-tokens"`) , and then use the theme to inject the proper color

iryanclarke replied:
Okay, sounds good!

iryanclarke reviewed (commented):
Just some Q's, otherwise looks 👌🏻

CCAyl commented on dist/tokens/native/core-light-colors.js:
```diff
@@ -10,4 +10,4 @@ purple : {100 : "#FCFAFF",200 : "#F2ECFE",300 : "#D7CEE9",400 : "#B9ABD5",500 :
 shades : {white : "#FFFFFF",black : "#000000",transparent : "transparent"},
-yellow : {100 : "#faf8f2",200 : "#fcf3dd",300 : "#fce7b1",400 : "#fcd97e",500 : "#fccb4c",600 : "#ffc121",700 : "#f7b200",800 : "#e5a500"},};
\ No newline at end of file
+yellow : {100 : "#faf8f2",200 : "#fcf3dd",300 : "#fce7b1",400 : "#fcd97e",500 : "#fccb4c",600 : "#ffc121",700 : "#f7b200",800 : "#e5a500"},boxShadows: {},};
```
still doesnt seem to b populating for native core files, we can look in a follow up if you wanna just get this in?

CCAyl commented on dist/tokens/ts/core-light-colors.d.ts:
```diff
@@ -10,4 +10,4 @@ export const purple : {100 : string,200 : string,300 : string,400 : string,500 :
 export const shades : {white : string,black : string,transparent : string};
-export const yellow : {100 : string,200 : string,300 : string,400 : string,500 : string,600 : string,700 : string,800 : string};
\ No newline at end of file
+export const yellow : {100 : string,200 : string,300 : string,400 : string,500 : string,600 : string,700 : string,800 : string};export const boxShadows: {};
```
hmm i think we have an issue with core all around?

alexandra-lim replied:
I forgot to mention I noticed this yesterday! We don't have any box shadow tokens for core so it's empty. I'll add a conditional within the formatter or look for a similar solution

CCAyl replied:
makes sense!! can do that in a follow up!
